lib/engine/game/g_18_neb/game.rb

```diff
@@ -375,7 +389,13 @@ class Game < Game::Base
             revenue: 5,
             desc: '$5 revenue each time ANY tile is laid or upgraded.',
             sym: 'P5',
-            # abilities: [{ type: 'blocks_hexes', owner_type: 'player', hexes: ['C7'] }],
+            abilities: [
+              {
+                type: 'tile_income',
+                terrain: nil,
```

Comment:
Don't include terrain in the configuration, its confusing. Not specifying will default to nil.

Reply:
ah, that makes sense. I've removed terrain.